src/components/Hero.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import heatmap from '../assets/table.png';
+import defaultSubreddit from '../helpers/constants';
+
+
+const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.h1` 
+  margin: 27px 0 9px 0;
+`;
+
+const SubTitle = styled.h2`
+  font-size: 16px;
+  font-weight: normal;
+`;
+
+const Subreddit = styled.h3`
+  font-size: 16px;
+  font-weight: 500;
+  margin: 15px -1px 36px 0;
```

Comment:
This is a similar situation as we have with the button below. I'd suggest to replace the -1px with 0. See my new comment there.